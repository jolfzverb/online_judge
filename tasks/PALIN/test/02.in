6
999
9999
122221
1222221
122220
1222220
