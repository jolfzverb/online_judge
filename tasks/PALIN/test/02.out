1001
10001
123321
1223221
122221
1222221
